openwisp_monitoring/check/tests/test_ping.py

```diff
@@ -24,15 +25,13 @@ class TestPing(TestDeviceMonitoringMixin, TransactionTestCase):
         '',
         bytes("fping: option requires an argument -- 'z'", encoding='utf8'),
     )
-    _FPING_OUTPUT = (
+    _FPING_UNREACHABLE = (
```

Comment:
I had moved `_FPING_OUTPUT` to `__init__.py` since it was required by three Test classes. This need didn't arose in the case of `_FPING_UNREACHABLE`, hence I hadn't moved it but I can move it.